unlock-app/src/components/interface/keychain/Key.tsx

```diff
@@ -300,10 +313,42 @@ function Key({ ownedKey, account, network }: Props) {
                         onClick={addToWallet}
                       >
                         <WalletIcon />
-                        Add to my wallet
+                        Add to my crypto wallet
                       </MenuButton>
                     )}
                   </Menu.Item>
+                  {isEthPassSupported(network) && (
+                    <>
+                      <Menu.Item>
+                        {({ active, disabled }) => (
+                          <MenuButton disabled={disabled} active={active}>
+                            <AddToGoogleWallet
+                              network={network}
+                              lockAddress={lock.address}
+                              tokenId={tokenId}
+                              image={metadata.image}
+                            />
+                          </MenuButton>
+                        )}
+                      </Menu.Item>
+                      <Menu.Item>
+                        {({ active, disabled }) => (
+                          <MenuButton disabled={disabled} active={active}>
+                            <AddToAppleWallet
```

Comment:
Perhaps add spacing? Is this intentional?

Reply:
I tried but for some reason I am not able to make these look nice :/

Reply:
Fixed! Thanks for your help!